### DISCUSSION TITLE
Matplotlib Line Display Question

### DISCUSSION THREAD

**Discussion by VeilleuxM** [cooperrc/advanced-dynamics#24] [Q&A]
Hello, 

For problem 1b of HW01, I am having issues with getting my code to display useful graphs. After running the script, a graph appears with the axes labeled with the appropriate numerical values, but no line graphics appear on the graph. Instead, several dozen lines of code appear in my output before the graph as such:

[<matplotlib.lines.Line2D at 0x7f9898a55910>,
 <matplotlib.lines.Line2D at 0x7f9898a55940>,
 <matplotlib.lines.Line2D at 0x7f9898a55a60>,
 <matplotlib.lines.Line2D at 0x7f9898a55b80>,
 <matplotlib.lines.Line2D at 0x7f9898a55ca0>,
 <matplotlib.lines.Line2D at 0x7f9898a55dc0>,
 <matplotlib.lines.Line2D at 0x7f9898a55ee0>,
 <matplotlib.lines.Line2D at 0x7f9898a5e040>,
 <matplotlib.lines.Line2D at 0x7f9898a5e160>,
                             ... 

![image](https://github.com/cooperrc/advanced-dynamics/assets/89992149/a52283f0-8cc7-4229-b5fc-e46eb4237f9b)

I am fairly confident in the code, which follows the example given earlier in HW01. Furthermore, I am able to view the arrays I am using for plt.plot() through a print() function, and the numerical values within the arrays make sense. For some reason, though, I am not able to plot these values on the graph. I have looked through several online forums and many recommend adding the plt.show() command at the end of my script. When I do this, the  <matplotlib.lines.Line2D> lines no longer appear in my output, but the graph still does not show the lines that I am looking for.

Has anyone else ran into a similar problem? Is there something obvious that I am missing that could fix the issue?

I appreciate any suggestions!

**Comment by liam-Wilson4** [comment]
One thing you may be able to try is to add a semi Colin to suppress the output of the values, and make sure you use  plt.

**Reply by VeilleuxM** [reply]
Thank you for the feedback! Unfortunately, I have done these things and it does not affect the output of the code.
